thoth/storages/data/alembic/versions/930b47e27b6c_rename_thoth_s2i_columns.py

```diff
@@ -0,0 +1,40 @@
+"""rename thoth s2i columns
+
+Revision ID: 930b47e27b6c
+Revises: b718c0f8bc53
+Create Date: 2022-01-19 16:56:54.439445+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '930b47e27b6c'
+down_revision = 'b718c0f8bc53'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('has_symbol', 'software_environment_id',
+               existing_type=sa.INTEGER(),
+               nullable=False)
+    op.alter_column('has_symbol', 'versioned_symbol_id',
```

Comment:
This change and the one below are not related to this even if created automatically.

Reply:
Actually the automated migration, didn't create a correct almebic migration.